dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java

```diff
@@ -48,8 +55,8 @@
 
     private static final String ENTERPRISE_WE_CHAT_TOKEN_URL = PropertyUtils.getString(Constants.ENTERPRISE_WECHAT_TOKEN_URL);
     private static final String ENTERPRISE_WE_CHAT_TOKEN_URL_REPLACE = ENTERPRISE_WE_CHAT_TOKEN_URL == null ? null : ENTERPRISE_WE_CHAT_TOKEN_URL
-    .replaceAll("\\{corpId\\}", ENTERPRISE_WE_CHAT_CORP_ID)
-    .replaceAll("\\{secret\\}", ENTERPRISE_WE_CHAT_SECRET);
+            .replaceAll("\\{corpId\\}", ENTERPRISE_WE_CHAT_CORP_ID)
+            .replaceAll("\\{secret\\}", ENTERPRISE_WE_CHAT_SECRET);
```

Comment:
```suggestion
            .replaceAll("\\{corpId}", ENTERPRISE_WE_CHAT_CORP_ID)
            .replaceAll("\\{secret}", ENTERPRISE_WE_CHAT_SECRET);
```
Please change the other part of `replaceAll`.